.github/workflows/ci.yml

```
@@ -18,6 +18,7 @@ env:
   # a step 'if env.AWS_USR' != ""', so we copy these to succinctly test whether
   # credentials have been provided before trying to run steps that need them.
   DOCKER_USR: ${{ secrets.DOCKER_USR }}
```

Applying this across the board and also to `DOCKER_PSW` should work here 👍🏻 
```suggestion
  CONTRIB_DOCKER_USR: ${{ secrets.CONTRIB_DOCKER_USR }}
```

---

Also will need to update https://github.com/crossplane-contrib/provider-argocd/blob/8fea55815aadd09c2c3f0bdbb2eaf323796608eb/Makefile#L58